src/GraphQL/Execution/ErrorInfoProviderOptions.cs

```diff
@@ -10,10 +10,30 @@ public class ErrorInfoProviderOptions
         /// <summary>
         /// Specifies whether stack traces should be serialized.
         /// </summary>
-        public bool ExposeExceptionStackTrace { get; set; }
+        [Obsolete("Use ExposeExceptionDetails property instead")]
+        public bool ExposeExceptionStackTrace
+        {
+            get => ExposeExceptionDetails;
+            set
+            {
+                ExposeExceptionDetails = value;
+                if (value)
+                    ExposeExceptionDetailsMode = ExposeExceptionDetailsMode.Message;
+            }
+        }
 
         /// <summary>
-        /// Specifies whether the extensions property, including by default the 'code', 'codes' and 'data' properties, should be serialized.
+        /// Specifies whether detailed exception information (exception types, stack traces, inner exceptions)
+        /// should be serialized.
+        /// </summary>
+        public bool ExposeExceptionDetails { get; set; }
+
+        /// <inheritdoc cref="Execution.ExposeExceptionDetailsMode"/>
+        public ExposeExceptionDetailsMode ExposeExceptionDetailsMode { get; set; }
```

Comment:
```suggestion
        public ExposeExceptionDetailsMode ExposeExceptionDetailsMode { get; set; } = ExposeExceptionDetailsMode.Extensions;
```

I thought we both agreed this would be better?  Note that with the code in the deprecated `ExposeExceptionStackTrace` property, this is a non-breaking change.

Reply:
ok